examples/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE HTML>
+<html>
+   <head>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a 'favicon' declaration in this 'header' tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a <title> tag to this page. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3APageWithoutTitleCheck)